numpy/lib/_arraysetops_impl.py

```diff
@@ -443,13 +438,10 @@ def unique_all(x):
 
     Examples
     --------
-    >>> import numpy as np
-    >>> np.unique_all([1, 1, 2])
-    UniqueAllResult(values=array([1, 2]),
-                    indices=array([0, 2]),
-                    inverse_indices=array([0, 0, 1]),
-                    counts=array([2, 1]))
-
+    >>> x = np.array([1, 1, 2])
+    >>> np.unique_all(x)
+    UniqueAllResult(values=array([1, 2]), indices=array([0, 2]),
+                    inverse_indices=array([0, 0, 1]), counts=array([2, 1]))
```

Comment:
Unforunately the spacing here matters - this is failing CI because `doctest` requires that the output from the call be character-identical with the REPL output.

Unfortunately again, fixing this is likely to cause a linter to complain due to the inherently long line length. That's okay though, passing doctests is more important.

Alternatively, the example could be recast to highlight the advantages of namedtuple without dumping all of the info on one line. For example:

```python
>>> x = [1, 1, 2]
>>> uniq = np.unique_all(x)
>>> uniq.values
array([1, 2])
>>> uniq.indices
array([0, 2])
```

... etc.